header-moh.php

```diff
@@ -21,40 +21,44 @@
 <title><?php wp_title( '|', true, 'right' ); ?></title>
 <link rel="profile" href="http://gmpg.org/xfn/11" />
 <link rel="pingback" href="<?php bloginfo( 'pingback_url' ); ?>" />
-<link href='http://fonts.googleapis.com/css?family=Open+Sans:300,400,400italic,600,600italic,700,700italic' rel='stylesheet' type='text/css'>
```

Comment:
Why remove the fonts?

Reply:
the fonts are loaded in functions.php, they are not needed in theme templates